common/app/views/fragments/containers/facia_cards/videoContainer.scala.html

```diff
@@ -0,0 +1,61 @@
+@(containerDefinition: layout.FaciaContainer)(implicit requestHeader: RequestHeader)
+
+@import model.VideoPlayer
+@import views.support.Video700
+@import views.html.fragments.media.video
+
+<div class="fc-container__inner">
+    <h2 class="video-title fc-container__header__title">
+        <a href="@containerDefinition.href">@containerDefinition.displayName</a>
+    </h2>
+</div>
+
+<div class="video-playlist video-playlist--0 js-video-playlist"
+     data-number-of-videos="@(containerDefinition.collectionEssentials.items.zipWithIndex.length - 1)">
+    <div class="video-playlist__control video-playlist__control--prev js-video-playlist-prev">
+        @fragments.inlineSvg("chevron-left", "icon", Seq("video-playlist__icon"))
+    </div>
+    <div class="video-playlist__control video-playlist__control--next js-video-playlist-next">
+        @fragments.inlineSvg("chevron-right", "icon", Seq("video-playlist__icon"))
+    </div>
+
+    <ul class="u-unstyled video-playlist__inner">
+        <li class="video-playlist__item video-title video-title--leftcol fc-container__header__title">
+            <a class="video-title__link" href="@containerDefinition.href">@containerDefinition.displayName</a>
+        </li>
+
+        @containerDefinition.collectionEssentials.items.zipWithIndex.map { case (item, index) =>
+            @if(item.header.isVideo) {
+                <li class="video-playlist__item video-playlist__item--@index">
```

Comment:
Could we not add a class `video-playlist__item--first` to avoid dynamic class names, and to be more explicit about what it i?